adr/ADR-6.md

```diff
@@ -28,27 +28,29 @@ A subject is comprised of 1 or more tokens. Tokens are separated by "." and can
 ## Specification
 
 ```
-dot              = "."
-asterisk         = "*"
-lt               = "<"
-gt               = ">"
-dollar           = "$"
-colon            = ":"
-double-quote     = ["]
-fwd-slash        = "/"
-backslash        = "\"
-pipe             = "|"
-question-mark    = "?"
-ampersand        = "&"
-dash             = "-"
-underscore       = "_"
-equals           = "="
-printable        = all printable ascii (33 to 126 inclusive)
-term             = (printable except dot, asterisk or gt)+
-limited-term     = (A-Z, a-z, 0-9, dash, underscore, fwd-slash, equals)+
-restricted-term  = (A-Z, a-z, 0-9, dash, underscore)+
-prefix           = (printable except dot, asterisk, gt or dollar)+
-filename-safe    = (printable except dot, asterisk, lt, gt, colon, double-quote, fwd-slash, backslash, pipe, question-mark, ampersand)
+dot                = "."
```

Comment:
We need to go away from all these possible scapes - the name is just an ID, and the ID should just be base64 - client can interpret use this anyway it fits its logic